gobblin-core/src/main/java/gobblin/commit/FsRenameCommitStep.java

```diff
@@ -131,6 +131,7 @@ public void execute() throws IOException {
       this.dstFs = getFileSystem(this.dstFsUri);
     }
     log.info(String.format("Moving %s to %s", this.srcPath, this.dstPath));
-    HadoopUtils.movePath(this.srcFs, this.srcPath, this.dstFs, this.dstPath, this.dstFs.getConf());
+    boolean overwrite = this.props.getPropAsBoolean(ConfigurationKeys.DATA_PUBLISHER_OVERWRITE_ENABLED, false);
```

Comment:
for this class can you set the `overwrite` property in the builder, since this `CommitStep` may not always be used in the publisher